src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -181,6 +182,8 @@ public enum SlackDistributionFailureBehavior {
 
     public static final String INCREMENTAL_TRANSFORMER_VOLTAGE_CONTROL_OUTER_LOOP_MAX_TAP_SHIFT_PARAM_NAME = "incrementalTransformerVoltageControlOuterLoopMaxTapShift";
 
+    public static final String INCREMENTAL_TRANSFORMER_REACTIVE_POWER_CONTROL_OUTER_LOOP_MAX_TAP_SHIFT_PARAM_NAME = "incrementalTransformerReactivePowerControlOuterLoopMaxTapShift";
```

Comment:
@jeandemanged can we shared the parameter with voltage control max shift?

Reply:
I guess we can yes

If we share, we should rename everywhere
INCREMENTAL_TRANSFORMER_VOLTAGE_CONTROL_OUTER_LOOP_MAX_TAP_SHIFT etc
to
INCREMENTAL_TRANSFORMER_TAP_CONTROL_OUTER_LOOP_MAX_TAP_SHIFT 

reviewing code I see that for phase shifter there is no max ... we should probably do something about it

Reply:
I'd say 
INCREMENTAL_TRANSFORMER_RATIO_TAP_CONTROL_OUTER_LOOP_MAX_TAP_SHIFT -> for both voltage and reactive power control by RTC

later on (**not in this PR**) we would introduce 

INCREMENTAL_TRANSFORMER_PHASE_TAP_CONTROL_OUTER_LOOP_MAX_TAP_SHIFT